DESCRIPTION

```diff
@@ -1,7 +1,6 @@
 Package: camtrapdp
 Title: Read and Manipulate Camera Trap Data Packages
-Version: 0.2.1.9000
-Date: 2024-06-05
```

Comment:
I know it's annoying (and against pkg recommendations), but please keep the `Date` property. It is required for a bunch of the generated metadata like cff

Reply:
What date would you like me to put here?